card-hoja1.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>CARD 1</title>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+<body>
+  <div class="book-cover">
+    <img class="badge" src="badge.png" alt="badge">
+    <img class="image-cover" src="book-cover.png" alt="Book cover" style="float:left;width:261px;height:368px;">
```

Comment:
Don't use the `style` attribute unless completely necessary to override the styles of an element. Also, you can use the `image-cover` class you already have 😉 . And remember to never ever use float, is really buggy and old way to position elements, try using flex in this element's container 😄

Reply:
Done